code/game/objects/items/storage/fancy.dm

```diff
@@ -269,6 +270,7 @@
 	throwforce = 2
 	w_class = SIZE_SMALL
 	flags_equip_slot = SLOT_WAIST
+	flags_obj = OBJ_IS_HELMET_GARB
```

Comment:
An entire cigar case doesn't need to be a helmet accessory (though its too long to fit anyways). You likely intended `/obj/item/clothing/mask/cigarette` (and thus subtype `/obj/item/clothing/mask/cigarette/cigar`) instead.

Reply:
resolved.